openstudiocore/src/shared_gui_components/MeasureManager.cpp

```diff
@@ -199,7 +201,8 @@ std::vector<ruleset::OSArgument> MeasureManager::getArguments(analysisdriver::Si
     if (appmodel)
     {
       model = appmodel;
-      idf = boost::none;

```

Comment:
You really need to see the two lines of code below 202 to see the action.  The github diff has been clipped.